packages/libcs50-git/libcs50-git.pacscript

```diff
@@ -0,0 +1,25 @@
+pkgname="libcs50-git"
+gives="libcs50"
+pkgver="11.0.3"
+source=("https://github.com/cs50/libcs50.git")
+makedepends=("make")
+maintainer=("Vaibhav Sharma <vvaibhavv3434@gmail.com>")
+pkgdesc="Library provided by Harvard University for learning the basic C language"
+
+build() {
+  cd "${_archive}"
```

Comment:
Please remove this line and anywhere else and replace it with something else, as this variable id deprecated.

Reply:
done changes